proto/cmp/types/v2/file.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+
+package cmp.types.v2;
+
+import "google/protobuf/timestamp.proto";
+
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1/file.proto.dot.xs.svg)
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1/file.proto.dot.svg)
+message File {
+  // file name
+  string name = 1;
+
+  // URL
+  string url = 2;
+
+  // Last modification time
+  google.protobuf.Timestamp last_modified = 3;
+}
+
+message Image {
+  // file
+  cmp.types.v2.File file = 1;
+
+  // Width
+  int32 width = 2;
+
+  // Height
+  int32 height = 3;
+
+  // Category
+  string category = 6;
+
+  // Image type
+  cmp.types.v2.ImageType type = 7;
+}
+
+message Video {
+  // File
+  cmp.types.v2.File file = 1;
+
+  // Codec of video
+  cmp.types.v2.Codec codec = 2;
+
+  // Bitrate in kbps
+  int32 bitrate = 3;
+
+  // Framerate
+  int32 framerate = 4;
+
+  // Width and height are specified in pixels. Aspect ratio can be derived from that
+  // Ex: "16:9"
+  int32 width = 5;
+  int32 height = 6;
+
+  // Container format. Ex: "MP4"
+  cmp.types.v2.VideoFormat format = 7;
+
+  // Category
+  string category = 8;
+}
+
+// Image formats
+enum ImageType {
```

Comment:
I think we should call this `ImageFormat` to be consistent with the video format below.